pages/property-filter/table.data.ts

```diff
@@ -16,18 +16,20 @@ export interface TableItem {
   releasedate?: Date;
   launchdate?: Date;
   lasteventat?: Date;
+  tags?: string[];
+  tagsIndex?: string;
 }
 
-export const states: Record<number, string> = {
-  0: 'Stopped',
-  1: 'Stopping',
-  2: 'Pending',
+export const states: Record<string, string> = {
```

Comment:
This to me looks like an `enum`. Any reason why we're not using an enum instead?

Reply:
This is not an enum - that is a mapping between the enum key like STOPPED, PENDING to a label. The keys can be made an enum but I see no much value in it as the mapping is still to exist.

Reply:
If it's a mapper then I don't think it would make sense as well.